chainfactory.go

```diff
@@ -42,16 +45,40 @@ type BuiltinChainFactory struct {
 	specs []*ChainSpec
 }
 
-// builtinChainConfigs is a mapping of valid builtin chain names
-// to their predefined ibc.ChainConfig.
-var builtinChainConfigs = map[string]ibc.ChainConfig{
-	"gaia":       cosmos.NewCosmosHeighlinerChainConfig("gaia", "gaiad", "cosmos", "uatom", "0.01uatom", 1.3, "504h", false),
-	"osmosis":    cosmos.NewCosmosHeighlinerChainConfig("osmosis", "osmosisd", "osmo", "uosmo", "0.0uosmo", 1.3, "336h", false),
-	"juno":       cosmos.NewCosmosHeighlinerChainConfig("juno", "junod", "juno", "ujuno", "0.0025ujuno", 1.3, "672h", false),
-	"agoric":     cosmos.NewCosmosHeighlinerChainConfig("agoric", "agd", "agoric", "urun", "0.01urun", 1.3, "672h", true),
-	"icad":       cosmos.NewCosmosHeighlinerChainConfig("icad", "icad", "cosmos", "photon", "0.00photon", 1.2, "504h", false),
-	"penumbra":   penumbra.NewPenumbraChainConfig(),
-	"composable": polkadot.NewComposableChainConfig(),
+//go:embed configuredChains.yaml
+var embeddedConfiguredChains []byte
+
+// initBuiltinChainConfig returns an ibc.ChainConfig mapping all configured chains
+func initBuiltinChainConfig() (map[string]ibc.ChainConfig, error) {
+
+	// checks if configuredChains.yaml is found in cwd or parent of cwd,
+	// otherwise it will use the the embedded version of configuredChains.yaml
+	configuredChainsFile := ""
+	if _, err := os.Stat("./configuredChains.yaml"); err == nil {
+		configuredChainsFile = "./configuredChains.yaml"
+	}
+	if _, err := os.Stat("../configuredChains.yaml"); err == nil {
```

Comment:
I'm not so sure about this. I would typically be very surprised about defaulting to a file in a single parent directory.

Reply:
As discussed on our call. Going to prioritize the embedded `configuredChains.yaml` unless an environment variable is present.